library/src/amd_detail/rocblaslt/src/rocblaslt_auxiliary.cpp

```diff
@@ -735,6 +739,8 @@ rocblaslt_status rocblaslt_matmul_desc_set_attribute(rocblaslt_matmul_desc
                     return rocblaslt_status_invalid_value;
                 }
                 break;
+            case ROCBLASLT_MATMUL_DESC_B_SCALE_POINTER_VEC_EXT:
+                matmulDesc->isScaleBVec = true;
             case ROCBLASLT_MATMUL_DESC_B_SCALE_POINTER:
                 if(sizeof(void*) <= sizeInBytes)
```

Comment:
If user set ROCBLASLT_MATMUL_DESC_B_SCALE_POINTER, matmulDesc->isScaleBVec should be reset to false.

Reply:
oh, right

Reply:
fixed